src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/messaging/PrisonerReceivedEventDueToTransferTest.kt

```diff
@@ -24,80 +24,22 @@ class PrisonerReceivedEventDueToTransferTest : IntegrationTestBase() {
   }
 
   @Test
-  fun `should update Review Schedule and send outbound message given 'prisoner received' (transfer) event for prisoner that has an active Review Schedule with review date later than adjusted date`() {
```

Comment:
We had 2 integration tests that were based on the original impl where the review date for a transfer was only updated if the new date was later than the existing review date.
When the behaviour was changed (so that review dates were always set to today+10 for transfers, regardless of existing review date) the 2 tests essentially became one and the same thing!

So I've tidied this up a little by deleting one of them, and removing the narrative about whether one date is later than another.